spec/features/categories_spec.rb

```diff
@@ -8,12 +8,12 @@
 
   let(:product) { Product.first }
   context "returns the Category selected", js: true do
-    it "returns the correct element" do
-      visit product_path(product)
+    it "returns the correct element", :correct_element do
```

Comment:
Rather than running a full integration that took about 5 minutes to complete every time I made changes, I decided to give this specific test case tag name `correct_element`